src/snake.h

```diff
@@ -53,6 +53,8 @@ namespace snake
         unsigned short int update();
         // destructor destroys game and ncurses
         virtual ~snake();
+        // if true you can't die from running in the opposite direction. bcs you can't run in the opposite direction.
```

Comment:
Comments are not for telling someone your whole life xD, something like "toogles possible movement into the opposite direction" is enaugh.